goal_src/jak3/engine/entity/entity.gc

```diff
@@ -2088,7 +2088,7 @@
     (iterate-process-tree *pusher-pool* s5-1 *null-kernel-context*)
     (iterate-process-tree *entity-pool* s5-1 *null-kernel-context*)
     )
-  (if #t
+  (if #f
```

Comment:
might be better to make stubs for these functions so we don't forget to fix it later